src/test/java/io/stargate/sgv2/jsonapi/api/v1/tables/CreateTableIntegrationTest.java

```diff
@@ -923,6 +923,38 @@ private static Stream<Arguments> allTableData() {
                   true,
                   SchemaException.Code.MISSING_DIMENSION_IN_VECTOR_COLUMN.name(),
                   "The dimension is required for vector columns if the embedding service is not specified.")));
+
+      // Two vector columns with the one has vectorizeDefinition and the other one doesn't
```

Comment:
Maybe add a note that this acceptable (only one distinct vectorize definition)? (this is implied by having `null` for failure, but just to keep things readable)